src/components/SuggestionsList/SuggestionsListItem.test.tsx

```diff
@@ -59,13 +59,20 @@ describe('<SuggestionsListItem />', () => {
   });
 
   describe('when selecting', () => {
+    let preventDefault: jest.Mock<{}>;
```

Comment:
`{}` is the default type argument, so no need to provide it.

Reply:
done

Reply:
my upcoming linting change will address the rest of these...